doc/links.txt

```diff
@@ -1,33 +1,39 @@
-.. _`Socket API`: https://pubs.opengroup.org/onlinepubs/9699919799/functions/socket.html
+.. _`nrfx`: https://github.com/NordicSemiconductor/nrfx/
 
-.. _`Beej's Guide to Network Programming`: https://beej.us/guide/bgnet/
+.. ### Links to Infocenter
 
 .. _`AT Commands Reference Guide`: https://infocenter.nordicsemi.com/topic/ref_at_commands/REF/at_commands/intro.html
 
 .. _`Credential storage management %CMNG`: https://infocenter.nordicsemi.com/topic/ref_at_commands/REF/at_commands/mob_termination_ctrl_status/cmng.html
 
 .. _`LTE Link Monitor`: https://infocenter.nordicsemi.com/topic/ug_link_monitor/UG/link_monitor/lm_certificate_manager.html
 
+.. _`nRF IEEE 802.15.4 radio driver documentation`: https://infocenter.nordicsemi.com/topic/struct_drivers/struct/radio_driver_latest.html
+
+.. ### Links to Nordic pages
+
 .. _`nRF9160 modem firmware zip file`: https://www.nordicsemi.com/Products/Low-power-cellular-IoT/nRF9160/Download#98C9E2578566420786ABD40B695FDB9B
 
-.. _`mbed TLS backend`: https://tls.mbed.org/
+.. ### Links to nrf (not a good way to link)
```

Comment:
`(not a good way to link)` ?

Reply:
Correct, links with `../../..` are not optimal. This comes from the rebase.